cluster/saltbase/salt/kube-proxy/kube-proxy.manifest

```diff
@@ -38,6 +38,14 @@ kind: Pod
 metadata:
   name: kube-proxy
   namespace: kube-system
+  # This annotation lowers the possibility that kube-proxy gets evicted when the
+  # node is under memory pressure, and prioritizes it for admission, even if
```

Comment:
@bprashanth Node eviction manager take actions on both memory and disk (space and inode). I know your current implementation only consider memory pressure condition, not include disk. I am ok as is here.